### DISCUSSION TITLE
clear_notification node in latest example

### DISCUSSION THREAD

**Discussion by sanderlv** [zachowj/node-red-contrib-home-assistant-websocket#487] [Q&A] [answered]
Hi, Sorry, I have coopied the nodes in my flows and they work well. Now I wanted to use the "clear_notification" as a node and as it is currently as an inject in the example I am truggling to get that in a node and thus in a flow:

![image](https://user-images.githubusercontent.com/16528916/137493305-ba4593f7-af60-4365-a669-9033bc108926.png)

![image](https://user-images.githubusercontent.com/16528916/137493359-d4c5dae9-fbb2-4254-b5ea-c9363738179b.png)

How do I get the "clear_notification" in a flow? What node?

**Comment by zachowj** [comment, marked as answer]
change node

**Comment by sanderlv** [comment]
so simple, so blind
